src/about-menu/open-about-menu.ts

```diff
@@ -27,11 +27,9 @@ export function openAboutWindow() {
 		},
 	} );
 
-	aboutWindow.loadFile( aboutPath );
-
 	// Open external links in the default browser
 	aboutWindow.webContents.setWindowOpenHandler( ( { url } ) => {
-		shellOpenExternalWrapper( url );
+		void shellOpenExternalWrapper( url );
```

Comment:
`void` is a way of intentionally signaling that no error handling is required for this promise. Most of the linter errors in our existing codebase can be handled this way, and it's the part I expect to be most controversial.